service/round.go

```diff
@@ -11,19 +18,16 @@ import (
 	"github.com/spacemeshos/poet/signal"
 	"github.com/spacemeshos/smutil/log"
 	"github.com/syndtr/goleveldb/leveldb/opt"
-	"os"
-	"path/filepath"
-	"sync"
-	"time"
 )
 
 type executionState struct {
-	NumLeaves     uint64
+	Epoch         uint32
 	SecurityParam uint8
 	Members       [][]byte
 	Statement     []byte
 	ParkedNodes   [][]byte
 	NextLeafID    uint64
+	NumLeaves     uint64
```

Comment:
i avoided that because there was a difference in how they were updated (NumLeaves after the end of the execution and NextLeadID was periodically persisted or if round was interrupted). but after a second look i think there are no downsides for merging it